01/components/Table.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import TableHeader from './TableHeader';
+import TableBody from './TableBody';
+import TableFooter from './TableFooter';
+
+class Table extends React.Component {
+    render() {
+        const { data } = this.props;
+        console.log(data);
+
+        const initialValue = 0
+        const total = data.reduce((acc, item) => acc + item.price * item.quantity, initialValue)
```

Comment:
Jeśli nie używasz dodatkowo nigdzie ` initialValue ` to napisałbym po prostu `data.reduce((acc, item) => acc + item.price * item.quantity, 0)`